stdlib/pervasives.mli

```diff
@@ -236,9 +236,16 @@ external ( @@ ) : ('a -> 'b) -> 'a -> 'b = "%apply"
 
 (** {1 Integer arithmetic} *)
 
-(** Integers are 31 bits wide (or 63 bits on 64-bit processors).
-   All operations are taken modulo 2{^31} (or 2{^63}).
-   They do not fail on overflow. *)
+(** Integers are [int_size] bits wide.
+    All operations are taken modulo 2{^[int_size]}.
+    They do not fail on overflow. *)
+
+val int_size : int
+(** Size of an integer, in bits. It is 31 (resp. 63) when using the OCaml
+    compiler on a 32-bit (resp. 64-bit) platform. It may differ for other
```

Comment:
Shouldn't we refer to the target machine/architecture rather than to compilers?

Reply:
How about this?
```ocaml
(** Size of an integer, in bits. It is 31 (resp. 63) when using OCaml on a
    32-bit (resp. 64-bit) platform. It may differ for other implementations,
    e.g. it can be 32 bits when compiling to JavaScript.
    @since 4.07.0
*)
```

Reply:
Seems fine. If you follow @gasche / @dra27 advice and do not
duplicate the value from `Sys`, you might consider updating the
documentation of `Sys`.